src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocMethodCheck.java

```diff
@@ -112,13 +112,35 @@
  * &lt;module name="MissingJavadocMethod"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   public Test() {} // violation, missing javadoc for constructor
+ *   public void test() {} // violation, missing javadoc for method
+ *   &#47;**
+ *    * Some description here.
+ *    *&#47;
+ *   public void test2() {}
+ * }
+ * </pre>
+ *
+ * <p>
  * To configure the check for {@code private} scope:
  * </p>
  * <pre>
  * &lt;module name="MissingJavadocMethod"&gt;
  *   &lt;property name="scope" value="private"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *   public void test() {} // violation, the public method is missing javadoc
```

Comment:
why here is violation if `scope` is set to private?

Reply:
I ran `MissingJavadocMethod` check on such an example. I threw a violation on methods with public scope too.

Reply:
This seems weird. In case this is a valid behavior, it should be added in description for config. If not, please create new issue.

Reply:
Well as specified in the documentation 
> Specify the visibility scope where Javadoc comments are checked.

`scope` must be checked only on the given files. So I don't think this is the intended behaviour. Should I create a issue for this?

Reply:
Yes. Please note that issue should follow issue template.

Reply:
Okayy

Reply:
@Gaurav-Punjabi please remove this line (public method and violation there) to avoid confusion for users

Reply:
Done.